scripts/kokoro/README.md

```
@@ -3,4 +3,5 @@
 Please see also
 https://huggingface.co/hexgrad/Kokoro-82M
 and
-https://huggingface.co/hexgrad/Kokoro-82M/discussions/14
+[https://huggingface.co/hexgrad/Kokoro-82M/discussions/14
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside link text

(MD039, no-space-in-links)